app/views/sites/redesigned_displays/_fonts_and_colors.html.haml

```diff
@@ -1,19 +1,13 @@
-%h3= t('.fonts_and_colors', scope: 'admin_center')
+%h3{:id => "fonts_and_colors"}= t('.fonts_and_colors', scope: 'admin_center')
 
--# NOTE: This is placeholder, disabled content. This subsection to be completed in SRCH-4142.
-= label_tag :font_family, t('.font_family', scope: 'admin_center')
-= select_tag :font_family,
-             options_for_select(%w(- Font_1 Font_2), selected: '-'), 
-             disabled: true
-
--# NOTE: This is placeholder, disabled content. This subsection to be completed in SRCH-4143.
-%fieldset
-  %legend= t('.color_scheme', scope: 'admin_center')
-  = f.label :theme, value: 'default', class: 'radio' do
-    = f.radio_button :theme, 'default', disabled: true
-    = t('.default', scope: 'admin_center')
-  = f.label :theme, value: 'custom', class: 'radio' do
-    = f.radio_button :theme, 'custom', disabled: true
-    = t('.custom', scope: 'admin_center')
+= f.fields_for :visual_design_json do |visual_design|
+  = visual_design.label :header_links_font_family, t('.header_links_font_family', scope: 'admin_center')
+  = visual_design.select :header_links_font_family,
+    options_for_select(Affiliate::USWDS_FONTS.index_by { |font| t(".fonts.#{font}", scope: 'admin_center') },
+    render_affiliate_visual_design_value(@site.visual_design_json, :header_links_font_family))
+  = visual_design.label :footer_and_results_font_family, t('.footer_and_results_font_family', scope: 'admin_center')
+  = visual_design.select :footer_and_results_font_family,
+    options_for_select(Affiliate::USWDS_FONTS.index_by { |font| t(".fonts.#{font}", scope: 'admin_center') },
+    render_affiliate_visual_design_value(@site.visual_design_json, :footer_and_results_font_family))
 
 %hr
```

Comment:
Really minor point: can we fix the missing new lines in this PR?

Reply:
Oh, absolutely yea. Thanks.